x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/cytoscape_options.tsx

```diff
@@ -65,6 +67,8 @@ function borderColorForNode(el: cytoscape.NodeSingular, theme: EuiThemeType) {
   const type = el.data('type');
 
   switch (type) {
+    case JOB_MAP_NODE_TYPES.ANALYTICS_JOB_MISSING:
+      return 'black';
```

Comment:
eui has a `#000` variable we can use
```suggestion
      return theme.euiColorFullShade;
```

Reply:
Updated in 031b09f27f9e9ba5e248fb79f6cd730b8c3b648b